patches/net/minecraft/client/renderer/block/model/ItemModelGenerator.java.patch

```diff
@@ -1,16 +1,6 @@
 --- a/net/minecraft/client/renderer/block/model/ItemModelGenerator.java
 +++ b/net/minecraft/client/renderer/block/model/ItemModelGenerator.java
-@@ -32,13 +_,16 @@
- 
-             Material material = p_111672_.getMaterial(s);
-             map.put(s, Either.left(material));
--            SpriteContents spritecontents = p_111671_.apply(material).contents();
```

Comment:
Maybe this is just github being weird but why is this patch being changed? It feels like potentially a messed up merge as it doesn't look like it has anything to do with networking.